src/idl_gen_swift.cpp

```diff
@@ -378,35 +377,6 @@ class SwiftGenerator : public BaseGenerator {
     code_ += "}\n";
   }
 
-  // Generates the create function for swift
-  void GenStructWriter(const StructDef &struct_def) {
-    const bool is_private_access =
-        parser_.opts.swift_implementation_only ||
-        struct_def.attributes.Lookup("private") != nullptr;
```

Comment:
nit: why is this deleted?

Reply:
This method is not used anywhere in the code.

Reply:
Perfect then! nice clean up! 🚀